source/Willow/DialogOpener.class.st

```diff
@@ -4,11 +4,15 @@ I help in opening dialogs. Useful outside of the DialogOpeningCommand.
 Class {
 	#name : #DialogOpener,
 	#superclass : #GRObject,
+	#pools : [
+		'Willow'
+	],
 	#category : #'Willow-WebInteraction'
 }
 
 { #category : #actions }
 DialogOpener >> open: aDialogView on: aCanvas [
 
-	^ (aCanvas jQuery id: WillowConstants dialogSectionName) append: [ :aSubCanvas | aSubCanvas render: aDialogView ]
+	^ (aCanvas jQuery id: Constants >> #willow >> #dialogSectionName)
```

Comment:
At least while reading the pull requests, it's to confusing that sometimes we write `Willow constants >> ...` and other times it's `Constants >> #willow`.
I'm almost certain they are not the same thing, but it's really confusing.